### DISCUSSION TITLE
How to migrate packages with all versions from other repository to verdaccio

### DISCUSSION THREAD

**Discussion by zuoez02** [verdaccio/verdaccio#4717] [Q&A]
I have a private repo with many versions packages and I want to migrate them to verdaccio, what should I do?

**Comment by juanpicado** [comment]
hi @zuoez02 I've heard this questions few times already, probably if you search you can get some insights how others are doing, but to let you know there is no a migration guidelines at this moment.

**Comment by zuoez02** [comment]
I have searched in the discussions, but I only got the answer about how to migrate from other verdaccio.
1. https://github.com/orgs/verdaccio/discussions/4699
2. https://github.com/orgs/verdaccio/discussions/3003

I want to use verdaccio to replace my cnpmjs. I can now only set an uplink to old cnpmjs. But actually I want only one verdaccio.
